From 7c22e4800ab5e17a1401c70ac08bcbc74cb5b55e Mon Sep 17 00:00:00 2001
From: George <31376482+george-gca@users.noreply.github.com>
Date: Fri, 29 Nov 2024 00:09:29 -0300
Subject: [PATCH 07/20] Fixed bug in header

---
 _includes/header.liquid | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/_includes/header.liquid b/_includes/header.liquid
index 7129e767..f25894e4 100644
--- a/_includes/header.liquid
+++ b/_includes/header.liquid
@@ -105,7 +105,7 @@
                   {%- elsif p.permalink contains '/blog/' -%}
                     {%- assign url = '/blog/' -%}
                   {%- else -%}
-                    {%- assign url = p['url-'] %}
+                    {%- assign url = p['url'] -%}
                   {%- endif %}
                   <a class="nav-link" href="{{ url | relative_url }}">
                     {{- p.title }}
-- 
2.48.1

